requirements.txt

```
@@ -16,6 +16,7 @@ wagtail==2.6.3
 wagtail-bakery==0.3.0
 whitenoise==4.1.4
 urlwait==0.4
+newrelic==5.2.1.129
```

Latest version has already (!) moved on

```suggestion
newrelic==5.2.2.130
```